test/HodlFactory.ts

```diff
@@ -43,16 +43,16 @@ describe('Hodl Factory Tests', function () {
 
   it('deploy a new clone', async() => {
 
-    const targetAddress = await factory.getTargetHTokenAddress(token.address, penalty, lockingWindow, expiry, fee, n, creator.address)
-    const deployedAddress = await factory.getCreatedHToken(token.address, penalty, lockingWindow, expiry, fee, n, creator.address)
+    const targetAddress = await factory.getTargetHTokenAddress(token.address, penalty, lockingWindow, expiry, fee, n, creator.address, '0x0000000000000000000000000000000000000000')
+    const deployedAddress = await factory.getCreatedHToken(token.address, penalty, lockingWindow, expiry, fee, n, creator.address, '0x0000000000000000000000000000000000000000')
```

Comment:
You can use `ethers.constants.AddressZero` instead of all the `0x0000000000000000000000000000000000000000`s!